tests/lipschitz_test.py

```diff
@@ -229,7 +96,10 @@ def test_stiefel_dense_mutable(self):
     # should be close to the distance between the points in the input space.
     self.assertAllClose(y1_y2_norm, x1_x2_norm, atol=atol, rtol=rtol)
 
-    self.assertAllClose(mutated_1['lip']['u'], mutated_2['lip']['u'], atol=atol, rtol=rtol)
+    parametrization_name = list(mutated_1['lip'].keys())[0]
+    lip_vars = mutated_1['lip'][parametrization_name]
+
+    self.assertAllClose(lip_vars['u'], lip_vars['u'], atol=atol, rtol=rtol)
```

Comment:
`lip_vars['u']` was compared with itself, I guess you meant to write:
```suggestion
    parametrization_name = list(mutated_1['lip'].keys())[0]
    lip_vars_1 = mutated_1['lip'][parametrization_name]
    lip_vars_2 = mutated_2['lip'][parametrization_name]
    self.assertAllClose(lip_vars_1['u'], lip_vars_2['u'], atol=atol, rtol=rtol)
```

Reply:
You're right, I had too much confidence in Github copilot this time.